src/_guides/language/effective-dart/design.md

```diff
@@ -1368,17 +1345,17 @@ that clarity.
 ### CONSIDER using the generalized function type syntax for parameters.
```

Comment:
I'd just say "function type syntax" or "inline function types".
The "generalized" makes no sense on its own, it's relative to something, and that something isn't clear here (if anywhere).